Templates/cmake/README.md

```diff
@@ -0,0 +1,41 @@
+CMake + Intel&reg; oneAPI Examples
```

Comment:
The Readme would need to be modified to match the existing template

Reply:
Oops, sorry, I made modifications but in a local copy, missed that out. I will update. Thanks for pointing.